test/runner.py

```diff
@@ -17,10 +17,6 @@
 TESTCASE_TIMEOUT = 100
 SCENARIOS_AT_ONCE = 100
 
-STDERR_ERR_PATTERNS = [
```

Comment:
Why is it deleted?

Reply:
This wasn't used anymore. Instead of fixing it in this PR I opted out for deleting it